charts/node-specific-sizing/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "knss.fullname" . }}-test-connection"
```

Comment:
The `knss` abbreviation seems to only be in use here and not part of `_helpers` as they are.

Reply:
lol yeah I first created the name chart with the name knss and then renamed it because i didn't like it. I'll correct that ^^